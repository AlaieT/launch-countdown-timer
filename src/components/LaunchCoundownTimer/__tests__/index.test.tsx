import React from "react";
import { render } from "@testing-library/react";

import LaunchCountdownTimer from "..";

describe("LaunchCountdownTimer testing...", () => {
  const props = {
    title: "WE'RE LAUNCHING SOON",
    deadline: "April 30, 2023",
    links: {
      facbook: "",
      pinterest: "",
      instagram: ""
    },
    background: {
      starsImage: "/",
      hillsImage: "/"
    }
  };

  it("Snapshot testing", () => {
    const { asFragment } = render(
      <LaunchCountdownTimer
        background={props.background}
        deadline={props.deadline}
        title={props.title}
        links={props.links}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
