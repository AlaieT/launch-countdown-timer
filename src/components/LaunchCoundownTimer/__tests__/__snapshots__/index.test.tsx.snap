// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LaunchCountdownTimer testing... Snapshot testing 1`] = `
<DocumentFragment>
  <header>
    <h1>
      WE'RE LAUNCHING SOON
    </h1>
  </header>
  <main>
    <div
      class="digit"
    >
      <div
        class="box"
      >
        <div
          class="top"
        >
          <p>
            21
          </p>
        </div>
        <div
          class="middle"
        >
          <div
            class="front"
          >
            <p>
              21
            </p>
          </div>
          <div
            class="back"
          >
            <p>
              21
            </p>
          </div>
        </div>
        <div
          class="bottom"
        >
          <p>
            21
          </p>
        </div>
      </div>
      <div
        class="label"
      >
        <p>
          days
        </p>
      </div>
    </div>
    <div
      class="digit"
    >
      <div
        class="box"
      >
        <div
          class="top"
        >
          <p>
            18
          </p>
        </div>
        <div
          class="middle"
        >
          <div
            class="front"
          >
            <p>
              18
            </p>
          </div>
          <div
            class="back"
          >
            <p>
              18
            </p>
          </div>
        </div>
        <div
          class="bottom"
        >
          <p>
            18
          </p>
        </div>
      </div>
      <div
        class="label"
      >
        <p>
          hours
        </p>
      </div>
    </div>
    <div
      class="digit"
    >
      <div
        class="box"
      >
        <div
          class="top"
        >
          <p>
            19
          </p>
        </div>
        <div
          class="middle"
        >
          <div
            class="front"
          >
            <p>
              19
            </p>
          </div>
          <div
            class="back"
          >
            <p>
              19
            </p>
          </div>
        </div>
        <div
          class="bottom"
        >
          <p>
            19
          </p>
        </div>
      </div>
      <div
        class="label"
      >
        <p>
          minuts
        </p>
      </div>
    </div>
    <div
      class="digit"
    >
      <div
        class="box"
      >
        <div
          class="top"
        >
          <p>
            07
          </p>
        </div>
        <div
          class="middle"
        >
          <div
            class="front"
          >
            <p>
              07
            </p>
          </div>
          <div
            class="back"
          >
            <p>
              07
            </p>
          </div>
        </div>
        <div
          class="bottom"
        >
          <p>
            07
          </p>
        </div>
      </div>
      <div
        class="label"
      >
        <p>
          seconds
        </p>
      </div>
    </div>
  </main>
  <footer>
    <ul>
      <li>
        <a
          aria-label="facebook link"
          href=""
          rel="noreferrer"
          target="_blank"
        >
          <div />
        </a>
      </li>
      <li>
        <a
          aria-label="pinterest link"
          href=""
          rel="noreferrer"
          target="_blank"
        >
          <div />
        </a>
      </li>
      <li>
        <a
          aria-label="instagram link"
          href=""
          rel="noreferrer"
          target="_blank"
        >
          <div />
        </a>
      </li>
    </ul>
  </footer>
  <div
    id="background"
  >
    <img
      alt="bg-stars"
      src="/"
    />
    <img
      alt="bg-hills"
      src="/"
    />
  </div>
</DocumentFragment>
`;
